docusaurus/src/components/HeaderDecoration.jsx

```diff
@@ -24,6 +22,65 @@ const CROSS_ICON = (
   </svg>
 );
 
+
+const MetadataStat = ({ label, children }) => (
+  <div className={styles.metadataStat}>
+    <dt className={styles.metadataStatLabel}>{`${label}: `}</dt>
+    <dd className={styles.metadataStatValue}>{children}</dd>
+  </div>
+);
+
+const EnabledIcon = ({ isEnabled }) => {
+  return isEnabled ? CHECK_ICON : CROSS_ICON;
+};
+
+const ConnectorMetadataCallout = ({ isCloud, isOss, isPypiPublished, supportLevel, github_url, dockerImageTag }) => (
+  <Callout className={styles.connectorMetadataCallout}>
+    <dl className={styles.connectorMetadata}>
+      <MetadataStat label="Availability">
+        <div className={styles.availability}>
+          <Chip className={isCloud ? styles.available : styles.unavailable}>
+            <EnabledIcon isEnabled={isCloud} /> Airbyte Cloud
+          </Chip>
+          <Chip className={isOss ? styles.available : styles.unavailable}>
+            <EnabledIcon isEnabled={isOss} /> Airbyte OSS
+          </Chip>
+          <Chip className={isPypiPublished ? styles.available : styles.unavailable}>
+            <EnabledIcon isEnabled={isPypiPublished} /> PyAirbyte
+          </Chip>
+        </div>
+      </MetadataStat>
+      <MetadataStat label="Support Level">
+        <a href="/integrations/connector-support-levels/">
+          <Chip>{supportLevel}</Chip>
+        </a>
+      </MetadataStat>
```

Comment:
nit: could we vertically center the support level chips so that they are vertically aligned with the `Support Level` label, similar to how the Availability chips are centered? 
<img width="517" alt="Screenshot 2024-05-16 at 2 27 10 PM" src="https://github.com/airbytehq/airbyte/assets/22731524/c93fe19f-53ed-4bde-9551-2037449d960e">